src/App.tsx

```diff
@@ -1,15 +1,43 @@
-import React from 'react';
-import './App.scss';
+import React, { useCallback, useState } from 'react';
+import debounce from 'lodash.debounce';
 import { peopleFromServer } from './data/people';
 
+import { Person } from './types/Person';
+
+import './App.scss';
+
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [activeDropDown, setActiveDropDown] = useState(false);
+  const [searchInput, setSearchInput] = useState('');
+  const [querry, setQuerry] = useState('');
+  const [selectedPerson, setSelectedPerson] = useState<Person | {}>({});
```

Comment:
Better set null as an initial value and just check if the value is not null

Reply:
Thank you!)